Classes/Controller/DocumentsController.php

```diff
@@ -83,9 +87,14 @@ public function indexAction(string $workspaceName = 'live'): void
         $siteNodeName = $site->getNodeName();
         foreach ($this->nodeEnumerator->siteNodeInContexts($site, $workspaceName) as $siteNode) {
             foreach ($this->nodeEnumerator->recurseDocumentChildNodes($siteNode) as $documentNode) {
-                if ($documentNode->getNodeType()->isOfType('Neos.Neos:Shortcut')) {
-                    continue;
+                $nodeType = $documentNode->getNodeType();
+
+                foreach ($this->getIgnoredNodeTypes() as $ignoredNodeType) {
```

Comment:
I would suggest to put this check into a method, so we can just return a boolean. That way we don't need the `continue 2`, which works but makes this harder to read.